src/unix/core.c

```diff
@@ -125,8 +125,8 @@ void uv_close(uv_handle_t* handle, uv_close_cb close_cb) {
     uv__check_close((uv_check_t*)handle);
     break;
 
-  case UV_IDLE:
-    uv__idle_close((uv_idle_t*)handle);
+  case UV_SPIN:
+    uv__spin_close((uv_spin_t *) handle);
```

Comment:
The change to the `*` is unrelated and violates the libuv style. Please change all instances of this back.